erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -493,6 +499,11 @@ def update_transaferred_qty_for_required_items(self):
 						'item': d.item_code
 					})[0][0]
 
+			required_qty = d.required_qty + (allowance_percentage/100 * self.qty)
```

Comment:
Allowance should not be checked for raw materials

Reply:
Fixed